app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,7 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.RECORD_AUDIO"/>
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
@shubertm For which reason do we need this permission?

Reply:
You can check the lint error here https://github.com/ARK-Builders/ARK-Memo/actions/runs/11668539341/job/32488691548. Seems glide posts some notifications. Yet am not really sure